### DISCUSSION TITLE
求助，我的绳网界面又出问题了，有人知道该怎么解决吗？

### DISCUSSION THREAD

**Discussion by 1155993355** [share121/inter-knot#1019] [其他]
变成这样了，哪位大佬能帮我解决一下？
![1000079821](https://github.com/user-attachments/assets/75f4a67e-f01a-4ac5-9231-9a419b33a781)
![1000079820](https://github.com/user-attachments/assets/f1118ccd-8a97-4a2e-b770-6dafc93bcae2)

**Comment by xingruxuewji** [comment]
屏幕缩放问题,看看手机能不能调整缩放比例.这个列数会根据你的缩放比例调整

**Reply by 1155993355** [reply]
我放大缩小了，但是还是这样子。
![Screenshot_2024-08-10-21-17-31-90_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/56ffdf1e-118d-4cef-b836-e7083363cb09)
![Screenshot_2024-08-10-21-17-26-25_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/cfd4354f-6494-45b5-b7cb-d818ef999f09)
![Screenshot_2024-08-10-21-17-17-67_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/d402de7b-e34c-40bb-be5e-f9e2146805a8)
![Uploading Screenshot_2024-08-10-21-17-11-05_a252b927494330cdc2c8ba3b3f952e5e.jpg…]()

**Comment by xingruxuewji** [comment]
或者你直接点击右下角三个横线,然后打开电脑模式

**Comment by CrazyFeng114** [comment]
手机右下角（或右上角）找桌面模式，打开就行了

**Comment by 1155993355** [comment]
各位问题解决了，我切换成电脑模式刷新了一下

**Comment by share121** [comment]
修复方法，见 https://github.com/share121/inter-knot/discussions/1100
